python/lsst/pipe/tasks/multiBand.py

```diff
@@ -697,6 +697,19 @@ class MergeMeasurementsConfig(MergeSourcesConfig):
     pseudoFilterList = ListField(dtype=str, default=["sky"],
                                  doc="Names of filters which may have no associated detection\n"
                                      "(N.b. should include MergeDetectionsConfig.skyFilterName)")
+    snName = Field(dtype=str, default="base_PsfFlux",
+                       doc="Name of flux measurement for calculating the S/N when choosing the "
+                           "reference band.")
```

Comment:
Can we justify the choices of default parameters, here and below? (Why PSF flux? why is ``minSN`` 10? why is ``minSNDiff`` 3?). Perhaps they're just what seems to work in practice, or perhaps there's more detailed consideration that the user should be aware of.

Since this is a port from HSC, if it's not straightforward to answer that question, we shouldn't block the merge but should consider filing a documentation debt ticket to come back to it later.

Also, please fix the indentation of ``doc`` (by matching it to ``dtype``, you can actually fit the whole ``doc`` argument on a single line).